postman/config.yml

```diff
@@ -0,0 +1,39 @@
+# Name of the quickstart (required)
+name: Postman
+
+# Displayed in the UI (required)
+title: Postman
+
+# Long-form description of the quickstart (required)
+description: |
+  Observe the performance of your APIs using Postman and New Relic
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Observe the performance of your APIs using Postman and New Relic
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Shashank Awasthi
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - API
+  - Postman
```

Comment:
```suggestion
  - postman
```

Reply:
I recommend lower case here, the search is actually case sensitive so it will be easier to find this way.